### ISSUE TITLE
About the final model

### ISSUE THREAD

**Issue by yongrenr** [f-kretschmer/bertax#18]
Hello, I am very interested in your work. I'd like to ask, is the final model"big_trainingset _all fix_classes_selection.h5" you gave trained with the training set and the test set together? We look forward to hearing from you

**Comment by f-kretschmer** [comment]
For the final model (as for all others), only the training partition of the corresponding dataset ("final model dataset") is used for training in order to get reliable metrics for its evaluation. See the [publication](https://doi.org/10.1073/pnas.2122636119), especially section "Three Evaluation Datasets" in the Methods for further details.
